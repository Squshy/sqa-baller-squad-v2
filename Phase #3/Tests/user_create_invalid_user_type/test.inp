CREATE
BIGBOIDADDYDUDE
password
ZZ
exit