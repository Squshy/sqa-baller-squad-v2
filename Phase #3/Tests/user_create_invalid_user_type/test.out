=========================================================
                        CREATE
=========================================================

Enter a username: BIGBOIDADDYDUDE

Enter a password: password

                    Available account types 

Full Standard (FS)  Buy Standard (BS)  Sell Standard (SS)

Enter an account type: ZZ

---------------------------------------------------------
                Invalid user type.
=========================================================